dmcontent/questions.py

```diff
@@ -112,7 +115,7 @@ def get_error_messages(self, errors, question_descriptor_from="label"):
         return question_errors
 
     def unformat_data(self, data):
-        return data
+        return {self.id: data[self.id]}
```

Comment:
Should this use `.get` by default in case the key does not exist in data?

For some routes, we call `unformat_data` on a question directly (https://github.com/alphagov/digitalmarketplace-supplier-frontend/blob/master/app/main/views/briefs.py#L202) and the data might not yet exist.